### DISCUSSION TITLE
blog/2023/social-directive/

### DISCUSSION THREAD

**Discussion by giscus** [choldgraf/choldgraf.github.io#69] [Blog comments]
# blog/2023/social-directive/

I often want to link to social and other types of web-based media in my Sphinx documentation and blog. Rather than embedding it all in custom HTML code, I decided to write a little wrapper to turn ...

https://chrisholdgraf.com/blog/2023/social-directive/

**Comment by 12rambau** [comment]
super useful, any chance that this get transformed into an pipy installable extention ?

**Reply by jpchauvel** [reply]
+1
